kuma/static/js/wiki-samples.js

```diff
@@ -94,7 +94,7 @@
                     // create icon
                     var $icon = $('<i />', { 'class': 'icon-' + sampleCodeHost, 'aria-hidden': 'true' });
                     // create text
-                    var $text = 'Open in ' + this + ' ';
+                    var $text = ngettext('Open in %(site)s ', this);
```

Comment:
I'm sorry, I'm new to the JS translation functions, and gave you bad advice.  ``ngettext`` is for picking between a singular and a plural text (like ``ngettext('View the record', 'View the records', record_count)``), but ``interpolate`` is used for populating a format string. I think this will work:

```
var $text = interpolate(gettext('Open in %(site)s '), {site: this});
```